docs/en/introduction/comparison/juicefs_vs_alluxio.md

```diff
@@ -29,48 +30,48 @@ The following table shows the differences of main features between Alluxio and J
 
 ## Storage format
 
-A single file is [stored](../architecture.md#how-juicefs-store-files) in JuiceFS in three levels: chunk, slice and block. A file will be split into multiple blocks, and be compressed and encrypted (optional) before it is stored into object storage.
+In JuiceFS, a single file is [stored](../architecture.md#how-juicefs-store-files) in three levels: chunk, slice, and block. The file is split into multiple blocks and is compressed and encrypted (optional) before it is stored in object storage.
```

Comment:
JuiceFS comes with its own storage format, which splits files into blocks, optionally encrypted and compressed before uploaded to the object storage. Read [how JuiceFS stores files](../architecture.md#how-juicefs-store-files) for more.

Reply:
Updated.